src/c/BuscaSequencialRecursiva.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+
+int buscaSequencialRecursiva(int vetor[], int i, int buscado, int size) {
+    
+    if (i == size)
+    {
+        return -1;
+    } 
+    else if (vetor[i] == buscado)
+    {
+        return i;
+    }
+    else
+    {
+        buscaSequencialRecursiva(vetor, i+1, buscado, size);
```

Comment:
A chamada recursiva precisa ser retornada:

```suggestion
        return buscaSequencialRecursiva(vetor, i+1, buscado, size);
```